packages/google_mobile_ads/android/src/test/java/io/flutter/plugins/googlemobileads/FlutterAdManagerInterstitialAdTest.java

```diff
@@ -185,6 +186,31 @@ public Object answer(InvocationOnMock invocation) throws Throwable {
     assertNull(flutterAdManagerInterstitialAd.getPlatformView());
   }
 
+  @Test
+  public void loadAdManagerInterstitialAd_setImmersiveMode() {
+    final AdManagerInterstitialAd mockAdManagerAd = mock(AdManagerInterstitialAd.class);
+    doAnswer(
+            new Answer() {
+              @Override
+              public Object answer(InvocationOnMock invocation) throws Throwable {
+                AdManagerInterstitialAdLoadCallback adLoadCallback = invocation.getArgument(3);
+                // Pass back null for ad
+                adLoadCallback.onAdLoaded(mockAdManagerAd);
+                return null;
+              }
+            })
+        .when(mockFlutterAdLoader)
+        .loadAdManagerInterstitial(
+            any(Context.class),
+            anyString(),
+            any(AdManagerAdRequest.class),
+            any(AdManagerInterstitialAdLoadCallback.class));
+
+    flutterAdManagerInterstitialAd.load();
+    flutterAdManagerInterstitialAd.setImmersiveMode(true);
+    verify(mockAdManagerAd).setImmersiveMode(anyBoolean());
```

Comment:
Can you check that it was `true`?

```suggestion
    verify(mockAdManagerAd).setImmersiveMode(eq(true));
```